backend/btrixcloud/pages.py

```diff
@@ -516,31 +516,42 @@ async def list_pages(
         page: int = 1,
         sort_by: Optional[str] = None,
         sort_direction: Optional[int] = -1,
+        include_total=False,
     ) -> Tuple[Union[List[PageOut], List[PageOutWithSingleQA]], int]:
         """List all pages in crawl"""
         # pylint: disable=duplicate-code, too-many-locals, too-many-branches, too-many-statements
         # Zero-index page for query
         page = page - 1
         skip = page_size * page
 
+        # Crawl or Collection Selection
+        if crawl_ids is None and coll_id is None:
+            raise HTTPException(
+                status_code=400, detail="either crawl_ids or coll_id must be provided"
+            )
+
+        if coll_id and crawl_ids is None:
+            crawl_ids = await self.coll_ops.get_collection_crawl_ids(
+                coll_id, public_or_unlisted_only
+            )
```

Comment:
Hm, I think if coll_id is provided, we should probably set crawl_ids to the crawls in that collection even if both arguments are provided, otherwise might get weird results?

Reply:
yeah, or could even error if both are provided, really should be one or the other.

Reply:
yup an HTTPException is even better!

Reply:
Updated!